src/Block.cpp

```diff
@@ -238,6 +238,7 @@ Block::~Block(void)
 	deleted_stms.clear();
 
 	local_vars.clear();
+	local_var_set.clear();
```

Comment:
I know it's a much larger scope than you originally proposed. The original data structure, vector, was chosen without consideration of performance. Thanks for making this right!